docs/user-manuals/multi-hierarchy-elastic-quota-management.md

```diff
@@ -0,0 +1,519 @@
+
+## Introduction
+When several users or teams share a cluster, fairness of resource allocation is very important. This proposal provides
+multi-hierarchy elastic quota management mechanism for the scheduler. 
+- It supports configuring quota groups in a tree structure, which is similar to the organizational structure of most companies.
+- It supports the borrowing / returning of resources between different quota groups, for better resource utilization efficiency.
+The busy quota groups can automatically temporarily borrow the resources from the idle quota groups, which can improve the
+utilization of the cluster. At the same time, when the idle quota group turn into the busy quota group, it can also automatically
+take back the "lent-to" resources.
+- It considers the resource fairness between different quota groups. When the busy quota groups borrow the 
+resources from the idle quota groups, the resources can be allocated to the busy quota groups under some fair rules.
+
+## Setup
+
+### Prerequisite
+
+- Kubernetes >= 1.18
+- Koordinator >= 0.71
+
+### Installation
+
+Please make sure Koordinator components are correctly installed in your cluster. If not, please refer to [Installation](/docs/installation).
+
+### Configurations
+
+Multi-Hierarchy-ElasticQuota-Management is *Enabled* by default. You can use it without any modification on the koord-descheduler config.
+
+## Use Multi-Hierarchy-ElasticQuota-Management
+
+### Quick Start
+
+1.Create a Deployment `quota-example` with the YAML file below.
+
+```yaml
+apiVersion: scheduling.sigs.k8s.io/v1alpha1
+kind: ElasticQuota
+metadata:
+  name: quota-example
+  namespace: default
+  labels:
+    quota.scheduling.koordinator.sh/parent: ""
+    quota.scheduling.koordinator.sh/is-parent: "false"
+spec:
+  max:
+    cpu: 40
+    memory: 40Gi
+  min:
+    cpu: 10
+    memory: 20Mi
+```
+
+```bash
+$ kubectl apply -f quota-example.yaml
+  elasticquota.scheduling.sigs.k8s.io/quota-example created
+
+$ kubectl get eqs -n default
+  NAME     AGE
+  test-d   2s
+```
+
+2.Create a pod `pod-example` with the YAML file below.
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-example
+  namespace: default
+  labels:
+    quota.scheduling.koordinator.sh/name: "quota-example"
+spec:
+  schedulerName: koord-scheduler
+  containers:
+  - command:
+    - sleep
+    - 365d
+    image: busybox
+    imagePullPolicy: IfNotPresent
+    name: curlimage
+    resources:
+      limits:
+        cpu: 40m
+        memory: 40Mi
+      requests:
+        cpu: 40m
+        memory: 40Mi
+    terminationMessagePath: /dev/termination-log
+    terminationMessagePolicy: File
+  restartPolicy: Always
+```
+
+```bash
+$ kubectl apply -f pod-example.yaml
+  pod/pod-example created
+```
+
+3.Verify `quota-example` has changed.
+```bash
+$ kubectl get eqs -n default quota-example -o yaml
+```
+```yaml
+kind: ElasticQuota
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
```

Comment:
remove the last-applied-configuration for better readability

Reply:
fixed